craft_application/fetch.py

```diff
@@ -130,11 +114,14 @@ def get_service_status() -> dict[str, Any]:
     return cast(dict[str, Any], response.json())
 
 
-def start_service() -> subprocess.Popen[str] | None:
-    """Start the fetch-service with default ports and auth."""
+def start_service() -> tuple[subprocess.Popen[str] | None, pathlib.Path]:
```

Comment:
This is *technically* an API change, but in something that isn't really what we consider to be part of the public API (though in practice it's not *not* public).